docs/user-guide/commands/iter8_k.md

```diff
@@ -39,6 +39,5 @@ Work with Kubernetes experiments
 * [iter8 k delete](iter8_k_delete.md)	 - Delete an experiment (group) in Kubernetes
 * [iter8 k launch](iter8_k_launch.md)	 - Launch an experiment inside a Kubernetes cluster
 * [iter8 k log](iter8_k_log.md)	 - Fetch logs for a Kubernetes experiment
-* [iter8 k report](iter8_k_report.md)	 - Generate report for Kubernetes experiment
 
 ###### Auto generated by spf13/cobra on 25-Jan-2023
```

Comment:
You appear to have hand edited these pages. They were automatically generated using the `iter8 doc` command. I am sure there are required parameters but I don't recall them.

Reply:
Yeah, I realize that these are generated. Just helped me make sure I got rid of all instances of `report`. I can revert?